frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1107,7 +1503,7 @@ function ReaderFooter:addToMainMenu(menu_items)
                     local SortWidget = require("ui/widget/sortwidget")
                     local sort_item
                     sort_item = SortWidget:new{
-                        title = _("Sort footer items"),
+                        title = _("Arrange items"),
```

Comment:
I prefer `Sort` but YMMV

Reply:
Another vote for sort :)

My non english nature might have something to do with it :D

Reply:
I do wonder if arrange might not imply a bit too much freedom. You can only adjust the order they're in. But the argument that sort somewhat implies alphabetical and friends isn't without merit either.

Reply:
`Reorder`?